### PR TITLE
Updating support for Adwords Conversion tracking

### PR DESCRIPTION
**Pull request by jasti**
Doc update

### CODE REVIEW

File: extensions/amp-analytics/amp-analytics.md [hunk 1]
```diff
@@ -156,6 +156,11 @@ Type attribute value: `gemius`
 
 Adds support for Gemius Audience/Prism analytics. Additionally, the gemius-provided `prefix` and `identifier` variables must be specified. It's also possible to specify the optional variable `extraparams` (key1=value1|key2=value2). More details can be found at [gemius.com](https://www.gemius.com).
 
+### Google Adwords
```
Comment (bpaduch) [inline extensions/amp-analytics/amp-analytics.md]: nit: In all occurrences of "Adwords", capitalize the "W" so it reflects the product name.

File: extensions/amp-analytics/amp-analytics.md [hunk 2]
```diff
@@ -156,6 +156,11 @@ Type attribute value: `gemius`
...
+
+Type attribute value: `googleadwords`
+Adds support for Google Adwords conversion tracking and remarketing. See more details in the AdWords help center for [conversion tracking](https://support.google.com/adwords/answer/1722054?hl=en) and [remarketing](https://support.google.com/adwords/answer/2453998?hl=en). Both tags can be used independent of each other.
```
Comment (bpaduch) [inline extensions/amp-analytics/amp-analytics.md]: Add a blank line between "Type" and the "Adds.." line so it looks consistent with the others.

File: extensions/amp-analytics/amp-analytics.md (hunk 1 above)
Comment (jasti) [inline extensions/amp-analytics/amp-analytics.md]: Done.

File: extensions/amp-analytics/amp-analytics.md (hunk 2 above)
Comment (jasti) [inline extensions/amp-analytics/amp-analytics.md]: Done.

**Pull request merged**